doc/users/whats_new/rcparams.rst

```diff
@@ -1,3 +1,9 @@
+Added ``xtick.minor.visible`` and ``ytick.minor.visible`` key to rcParams
+`````````````
+Two new keys to control the minor ticks on x/y axis respectively, default set to False(hide minor ticks on the axis).
```

Comment:
It is less 'hide' and more 'never shown' as by the current default is a `NullLocator`